src/controllers/commentController.js

```diff
@@ -0,0 +1,53 @@
+import Response from '../utils/response';
+import CommentService from '../services/commentService';
+
+const { createComment, getComment, deleteComment } = CommentService;
+const { success, badRequest } = Response;
+/**
+ * Controller form request comments
+ */
+export default class CommentController {
+/**
+ * @param {object} res - response object
+ * @return {object} user - return object containing status and data
+ */
+  static async postComment({ body: { comment }, params: { requestId }, user: { id } }, res) {
+    try {
+      const data = await createComment(comment, requestId, id);
```

Comment:
You can just pass the req object to the getComment then destructure it from the commentService getComment method.